test/noop-set.test.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const tap = require('tap')
+const noopSet = require('../lib/noop-set')
+
+tap.test('does a lot of nothing', async t => {
+  const aSet = noopSet()
+  t.type(aSet, 'object')
+
+  const item = {}
+  aSet.add({ another: 'item' })
```

Comment:
```suggestion
  aSet.add(item)
  aSet.add({ another: 'item' })
```

My bad, I deleted a line I did not want to delete